cmd/rancher_release/check_rancher_rc_deps.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"errors"
+
+	"github.com/rancher/ecm-distro-tools/release/rancher"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+func checkRancherRCDepsCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "check-rancher-rc-deps",
+		Usage: "check if the Rancher version specified by the commit or pre-release title does not contain development dependencies and rc tags",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
```

Comment:
Does this need to be required? The program changes behavior if it running inside the git repository, and only some files respect the commit hash, and other files use the local copy. Is there a way to make it use only local files when not specified, and only remote files when it is?